Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.h

```diff
@@ -259,10 +245,8 @@ ALLOW_NEW_API_WITHOUT_GUARDS_END
     void notifyActiveSourceBuffersChanged() override;
 
     void setPresentationSize(const IntSize&) final;
-    void setVideoLayerSizeFenced(const FloatSize&, WTF::MachSendRightAnnotated&&) final;
 
-    void updateDisplayLayer();
-    RefPtr<VideoMediaSampleRenderer> layerOrVideoRenderer() const;
+    Ref<AudioVideoRenderer> audioVideoRenderer() const { return m_renderer; }
```

Comment:
This can be moved out-of-line if it allows you to remove the `#include "AudioVideoRendererAVFObjC.h"` above. It's `private:` so there's no benefit to inlining.

Reply:
it's used by the SourceBuffer which are "friends"